### DISCUSSION TITLE
error on incoming webhook

### DISCUSSION THREAD

**Discussion by hafriz** [LiveHelperChat/livehelperchat#1858] [Q&A] [answered]
hi, i have and error on incoming webhook setup

`[message:protected] => Argument 2 passed to erLhcoreClassChatWebhookIncoming::processEvent() must be of the type array, null given, called in /web/livechat/lhc_web/cache/compiledtemplates/4242ff6c00f157807d8ee5a925afa6f8.php on line 2`

anyone know how to solve it?

**Comment by remdex** [comment, marked as answer]
Wrong payload. It has to be valid payload to pass :)

**Reply by hafriz** [reply]
Thanks remdex, i try to test it using postman. may i know why incoming webhook not working with whatsapp business api? it's can't validate i'm using fbmessenger plugin

**Reply by remdex** [reply]
It does work with all my known instalattions :) it looks more like your server does not support json payload request

**Comment by hafriz** [comment]
i will try to install in another server
